packages/blockchain-wallet-v4-frontend/src/data/components/debitCard/slice.ts

```diff
@@ -2,12 +2,15 @@ import { createSlice, PayloadAction } from '@reduxjs/toolkit'
 
 import { Remote } from '@core'
 
-import { CardActionType, DebitCardState, DebitCardType, ProductType } from './types'
+import { AccountType, CardActionType, DebitCardState, DebitCardType, ProductType } from './types'
 
 const initialState: DebitCardState = {
   cardCreationData: Remote.NotAsked,
   cardToken: '',
-  cards: [],
+  cards: Remote.NotAsked,
+  currentCardAccount: Remote.NotAsked,
+  currentCardSelected: undefined,
+  eligibleAccounts: [],
```

Comment:
```suggestion
  eligibleAccounts: Remote.NotAsked,
```

Reply:
Eligible accounts is not Remote type, and will not be at least in this commit as is not needed too